dev/run-tests-jenkins.py

```diff
@@ -183,6 +183,8 @@ def main():
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop2.7"
     if "test-hadoop3.2" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop3.2"
+    if "test-java11" in ghprb_pull_title:
```

Comment:
I might be forgetting, but I'm not sure we have set up all the necessary changes for cross compilation yet. Building with Java 11 may cause problems. Testing with Java 11 is all we want to do.

Reply:
Oh, I thought we set up roughly necessary things. We currently build against 8 and test against JDK 11 in Jenkins.

JDK11 build seems being tested via github workflow (https://github.com/apache/spark/runs/267887669)
I think we don't explicitly run JDK 11 build and JDK 11 tests yet if I didn't miss anything.

Reply:
Maybe I'm not remembering that it was resolved - do we compile with `-release 8` now? I don't see that in the build and recall there was some issue with it previously, but maybe I'm forgetting. Either way I don't think it's necessary to set the compilation JDK here?

Reply:
Ah, -release 8 wasn't resolved IIRC